glom/core.py

```diff
@@ -1379,6 +1401,69 @@ def _t_eval(target, _t, scope):
 ROOT = make_sentinel('ROOT')
 
 
+def _format_t(path, root=T):
+    prepr = ['T' if root is T else 'S']
+    i = 0
+    while i < len(path):
+        op, arg = path[i], path[i + 1]
+        if op == '.':
+            prepr.append('.' + arg)
+        elif op == '[':
+            prepr.append("[%s]" % (bbrepr(arg),))
+        elif op == '(':
+            args, kwargs = arg
+            prepr.append(format_invocation(args=args, kwargs=kwargs, repr=bbrepr))
+        elif op == 'P':
+            return _format_path(path)
+        i += 2
+    return "".join(prepr)
+
+
+class Val(object):
```

Comment:
Should we make `Literal` an alias to this? Seems a bit odd to have two virtually identical types?

Reply:
sure just didn't want to kill the docstring + etc for `Literal` -- also the "meat" of the spec is literally just `return self.value`